perf/lua/CMakeLists.txt

```diff
@@ -67,8 +67,9 @@ endfunction()
 create_perf_lua_test(NAME 1mops_write)
 create_perf_lua_test(NAME box_select)
 create_perf_lua_test(NAME gh-7089-vclock-copy)
-create_perf_lua_test(NAME uri_escape_unescape)
 create_perf_lua_test(NAME luafun)
+create_perf_lua_test(NAME tuple_field_by_name)
```

Comment:
We discussed it in a private conversation and I also left a comment here.
Probably we should organize our microbenchmarks into groups and filename of this bench should be renamed to `tuple.lua`, this will allow us to add other microbenchmarks related to tuples to the same file. Microbenchmarks in a single file are reported as a single test to Grafana.

Reply:
I suppose it may be done later, when we have a variety of the benchmarks to analyze.
Obviously, the Lua benchmarks should stay separate from the C ones. Also, it is a good idea to separate Lua benchmarks by the module (maybe?).
But for now, for simplicity, let's just use the microbenchmark for the specific target.

Reply:
I would rename a file to `tuple` to make a name more general. But I don't want to insist.